Add regex to aggregate equation operators

SkylineThomasCR opened the pull request:
This operator was introduced in 10.1.8 by RN30199.
See https://docs.dataminer.services/develop/devguide/Connector/Actions/ActionAggregate.html#equation

SkylineThomasCR commented:
6247f43948998db359d072314c6c5f71cdbb0b21
@MichielOda, I've added a new ValidatorResult. Which ID should it have? I used 9999 as placeholder.

MichielOda commented:
> [6247f43](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.Validators/commit/6247f43948998db359d072314c6c5f71cdbb0b21) @MichielOda, I've added a new ValidatorResult. Which ID should it have? I used 9999 as placeholder.

I would suggest to skip the error message. If this needs to be improved, the check needs to be converted to the new format. The intention is not to introduce new error messages in the legacy code, only fixes.

SkylineThomasCR commented:
> > [6247f43](https://github.com/SkylineCommunications/Skyline.DataMiner.CICD.Validators/commit/6247f43948998db359d072314c6c5f71cdbb0b21) @MichielOda, I've added a new ValidatorResult. Which ID should it have? I used 9999 as placeholder.
> 
> I would suggest to skip the error message. If this needs to be improved, the check needs to be converted to the new format. The intention is not to introduce new error messages in the legacy code, only fixes.

Removed new error message in cb5534e8ed9500954b8c277a043c827011bc501d
